Switch to YAML issue templates (#4971)
cython/cython: .github/ISSUE_TEMPLATE/other.md

---
name: Other

> about: Anything that doesn't neatly fit into the "bug" or "feature request" templates. DO NOT post support requests here.

Reviewer: At first glance, it is read as `Anything that doesn't neatly fit into the "bug" or "feature request"`
```suggestion
about: Anything that doesn't neatly fit into the listed templates. DO NOT post support requests here.
```

Author: I'm not completely convinced this is clearer so have left it for now

> about: Anything that doesn't neatly fit into the "bug" or "feature request" templates. DO NOT post support requests here.

Reviewer 2: ```suggestion
about: Anything that does not qualify as either "bug" or "feature request". DO NOT post support requests here.
```
